Processes/Docs-Contribution-Process-Internal.md

```diff
@@ -0,0 +1,105 @@
+
+
+# Docs Contribution Process - Internal
+
+## Overview
+This document provides instructions for contributors internal to the ENF Foundation on how to contribute new or updated content to EOS Documentation (https://docs.eosnetwork.com).
+
+ You can rely on the DevRel team to perform a thorough edit and review of your content before it is published to the EOS Documentation portal. The publication process includes GitHub workflows to publish Markdown (.md) formatted content.
+
+As a content contributor you are responsible for the accuracy and completeness of your submitted documentation. The DevRel team may contact you for clarification of your contribution, if necessary. You are also responsible to identify future changes for your content and let the DevRel team know of any information maintenance issues.
+
+As a content contributor you are also responsible to review the final submission before it is published. Your review is reflective of the accuracy and completeness of the content.
+
+### When to Use this Procedure
+Use this procedure when you need to create or update content from one or repositories to  [EOS Documentation](https://docs.eosnetwork.com).
+
+### Expected Results
+Fresh, updated content is available for consumption in EOS Documentation.
+
+### Stages of Publication with Contributor Roles Identified
+1. Author: Create branch
```

Comment:
Why do these all say "Author"?

Reply:
"Author" is standard language for the person who writes the content. I removed it.

Done